### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by LeifParr** [petrobras/ross#758] [Q&A] [answered]
Good evening folks in the ROSS community. There has been a good deal of rooting out bugs and adding new features since version 0.4.1. Does any one know when a new official release is planned?

**Comment by raphaeltimbo** [comment, marked as answer]
Hi there @LeifParr!
We are planning on releasing a 1.0 version of ROSS probably next week; we are just trying to improve documentation before releasing it.
In the meantime, if you need the updated code you could have the development version installed following [these instructions](https://ross-rotordynamics.github.io/ross-website/contributing.html#how-to-contribute-to-ross-using-git).

**Reply by LeifParr** [reply]
Ok. Thanks for info. I think I will stick with my 0.4.1 version until after wrapping up my current project, which I am about to finish.
